src/odemis/driver/pmtctrl.py

```diff
@@ -447,6 +470,36 @@ def _sendCommand(self, cmd):
 
             return ans.rstrip()
 
+    def _tryRecover(self):
+        self._recovering = True
+        self.state._set_value(HwError("PMTControl disconnected"), force_write=True)
+        # Retry to open the serial port (in case it was unplugged)
+        # _ser_access should already be acquired, but since it's an RLock it can be acquired
+        # again in the same thread
+        with self._ser_access:
+            while True:
+                try:
+                    self._serial.close()
+                    self._serial = None
+                except Exception:
+                    pass
+                try:
+                    logging.debug("retrying to open port %s", self._port)
```

Comment:
As typically `port` is a pattern (eg, /dev/PMT*), this message is a little odd. I'd suggest something like "Searching for the device on port %s".